Update issue templates [skip-ci] (#6330)
root-project/root: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for ROOT
title: ''
labels: new feature
assignees: Axel-Naumann

---

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**

> A clear and concise description of any alternative solutions or features you've considered.

Reviewer: ```suggestion
Can you think of alternative solutions or features?
```
3 times "clear and concise`

**Additional context**

> Add any other context or screenshots about the feature request here.

Reviewer: ```suggestion
Add any other context or screenshots about the feature requested here.
```
